slick.grid.js

```diff
@@ -1441,18 +1455,22 @@ if (typeof Slick === "undefined") {
     }
 
     function applyColumnWidths() {
-      var x = 0, w, rule, canvas;
+      var x = 0;
       for (var i = 0; i < columns.length; i++) {
-        w = columns[i].width;
-        rule = getColumnCssRules(i);
-        rule.left.style.left = x + "px";
-        canvas = i > options.pinnedColumn ? contentCanvas[1].width : contentCanvas[0].width;
-        rule.right.style.right = (canvas - x - w) + "px";
+        var column = columns[i];
+        var width = getColumnVisibleWidth(column);

```

Comment:
I generally like `var`s closer to where they're used.  Would you prefer me to stick to all `var`s at the top?